org.cloudcompare.CloudCompare.yaml

```diff
@@ -11,6 +13,8 @@ finish-args:
   - --share=ipc
   - --device=dri
   - --share=network
+  - --filesystem=host
```

Comment:
Looking at the CI failure (https://buildbot.flathub.org/#/builders/26/builds/11656), this makes the `--filesystem=home` specification below redundant. Do we need full access to the host filesystem? Most things should work with the "portals", which let users decide via a GUI from where to load files from. AFAIK, providing `--filesystem=` is only required when an application is supposed to load data on its own. This may apply, e.g., if the texture for a user-provided mesh has to be loaded afterwards.

Reply:
no we don't need it, your are right. It's an artifact of an old test.